javascript-objects/main.js

```diff
@@ -0,0 +1,37 @@
+var student = {
+  firstName: 'Heesoo',
+  lastName: 'Kim',
+  age: '24'
+};
+
+var fullName = student.firstName + student.lastName;
+console.log('the value of fullName:', fullName);
+
+student.livesInIrvine = true;
+student.previousOccupation = 'student';
+
+console.log('the value of student.livesInIrvine:', student.livesInIrvine);
+console.log('the value of student.previousOccupation:', student.previousOccupation);
+
+var vehicle = {
+  make: 'Tesla',
+  model: 'Model S',
+  year: '2021'
+};
+
+vehicle.color = 'white';
+vehicle.isCovertible = false;
+
+console.log('the value of vehicle.color:', vehicle.color);
+console.log('the value of vehicle.isConvertible:', vehicle.isConvertible);
```

Comment:
Please use square bracket notation, as requested in the directions
![image](https://user-images.githubusercontent.com/32880187/123709632-3b481700-d822-11eb-9c79-f85d1d49d317.png)